source/Nevermore.Tests/Util/LiteralStringIdDataModificationQueryBuilderFixture.cs

```diff
@@ -0,0 +1,444 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Assent;
+using Nevermore.AST;
+using Nevermore.Contracts;
+using Nevermore.Mapping;
+using Nevermore.Util;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace Nevermore.Tests.Util
+{
+    public class LiteralStringIdDataModificationQueryBuilderFixture
```

Comment:
This is just a rename of the previous `DataModificationQueryBuilderFixture`, using the old style (`string` ID) documents.